topic-operator/src/main/java/io/strimzi/operator/topic/InvalidTopicException.java

```diff
@@ -6,6 +6,10 @@
 
 import io.fabric8.kubernetes.api.model.HasMetadata;
 
+/**
+ * Used when an invalid topic is detected
```

Comment:
Small niggle, you have consistently used the term `Thrown when` but used `Used when` here. Won't block approval ofcourse, but might be worth having consistent comments.